sysutils/salt/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem        1.0
 
 name               salt
-version            2019.2.0
```

Comment:
the whitespace doesn't follow the modeline; please correct this in a separate commit not mixed with functional changes